.github/workflows/ci_cron.yml

```diff
@@ -1,51 +1,57 @@
-
-name: Weekly cron
+name: test on schedule
 
 on:
   schedule:
     # Weekly Monday 6AM build
     - cron: "0 0 * * 1"
```

Comment:
Comment line says 6AM, but the use of the "0 0" arguments means midnight (00:00), right?